### PR TITLE
add feedback icon and config

### LINKED ISSUE #2528: Implement Feedback Form as a feature
As [Sebastian](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#sebastian---tech-lead) and [Mathew](https://github.com/kyma-project/community/blob/main/manifesto/personas.md#matthew---cloud-native-software-engineer) I would like to be able to provide feedback to Busola Team so that improvements can be captured.

**Acceptance Criteria**

- [ ] Follow Project DOD
- [ ] Add icon and survey opens in a new tab.
- [ ] It's implemented as via feature flag
- [ ] the survey link is part of the feature

**Useful Links**

- [example](https://example.com)

Image

**Links** 

- [Design Specification in Figma](https://www.figma.com/file/AkzF9I1INdwc5iUtm5fKNU/Kyma?type=design&node-id=4978%3A106629&mode=design&t=xJvLT795RAgIiIqK-1)

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**

Changes proposed in this pull request:

- add feedback icon,
- add feature flag.

**Related issue(s)**
Closes #2528

### CODE REVIEW

**Review by akucharska — COMMENTED** [review]
Can we add it in documentation: `docs.features.md`

**Review by akucharska — DISMISSED** [review]

File: docs/features.md [hunk 1]
```diff
@@ -262,3 +262,13 @@ The **match** keys and **messageSrc** must use the format described in the [`jso
       policies:
         - Default
   ```
+
+- **FEEDBACK** - determines if feedback icon should be rendered at top bar and link when user will be redirected.
```
Comment (IwonaLanger) [inline docs/features.md:266]:
```suggestion
- **FEEDBACK** - determines if the feedback icon should be rendered at the top bar and link when the user is redirected.
```

**Review by akucharska — DISMISSED** [review]

**Review by IwonaLanger — APPROVED** [review]

**Review by akucharska — APPROVED** [review]

**Pull request merged**